server/eventlog.go

```diff
@@ -425,8 +425,8 @@ func getGrubState(hash crypto.Hash, events []*pb.Event) (*pb.GrubState, error) {
 			continue
 		}
 
-		if event.GetUntrustedType() != IPL {
-			return nil, fmt.Errorf("invalid event type for PCR%d, expected EV_IPL", index)
+		if event.GetUntrustedType() != IPL && event.GetUntrustedType() != EventTag {
```

Comment:
Should we explicitly allow/ignore "EventTag" only on PCR9?

Here if PCR is 8 it'll not throw an error and may fail later in this loop.

@alexmwu

Reply:
I think what we have here is fine. according to the PFP (https://trustedcomputinggroup.org/wp-content/uploads/TCG-PC-Client-Platform-Firmware-Profile-Version-1.06-Revision-52_pub-2.pdf), EV_EVENT_TAG is "Used for PCRs defined for OS and
application usage." Which are PCRs 8 and higher. https://github.com/systemd/systemd/blob/main/docs/TPM2_PCR_MEASUREMENTS.md uses it for PCR 12.

Reply:
Oh I see your point. This shouldn't parse the EV_EVENT_TAG event, since it likely comes from Linux. Right now we put the digest of the EV_EVENT_TAG event into the GRUB files, which isn't right

Reply:
So, put another way. If we see an EV_EVENT_TAG, we should both not throw an error AND skip this event.

Reply:
Change the logic a bit to skip parsing EV_EVENT_TAG